### ISSUE TITLE
dark mode 

### ISSUE THREAD

**Issue by xxcriticxx** [piejanssens/premiumizer#300]
can we request dark mode feature?

**Comment by neox387** [comment]
Yes this is a good Idea, will have a look when I find time altough htlm stuff is a bit unknown to me =)

Quick Google does give suggestions:
https://betterprogramming.pub/a-better-approach-to-dark-mode-on-your-website-dadbe8c55b40

Maybe also possible trough some flask themes package.

**Comment by xxcriticxx** [comment]
bad bot!

**Comment by JRDevo** [comment]
Been playing around with Stylus (browser extension) and made this: https://userstyles.world/style/7706/premiumizer-dark.

Seems to be a good alternative and also works with other apps like Sonarr.
